src/windows/enumerate.rs

```diff
@@ -287,7 +290,7 @@ impl PortDevice {
 
         let mut port_name_buffer = [0u16; MAX_PATH];
         let mut port_name_len = port_name_buffer.len() as DWORD;
-        let value_name: Vec<u16> = "PortName".encode_utf16().chain(Some(0)).collect();
+        let value_name = as_utf16("PortName");
 
         unsafe {
             RegQueryValueExW(
```

Comment:
While we are here: Shouldn't we check the type returned through [`RegQueryValueExW`](https://learn.microsoft.com/de-de/windows/win32/api/winreg/nf-winreg-regqueryvalueexw)'s `lpType` as for `RegEnumValueW`?